cmd/juju/machine/add.go

```diff
@@ -246,29 +256,11 @@ func (c *addCommand) Run(ctx *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
-	if c.Placement != nil && c.Placement.Scope == "ssh" {
-		logger.Infof("manual provisioning")
-		authKeys, err := common.ReadAuthorizedKeys(ctx, "")
-		if err != nil {
-			return errors.Annotate(err, "reading authorized-keys")
-		}
-		args := manual.ProvisionMachineArgs{
-			Host:           c.Placement.Directive,
-			Client:         client,
-			Stdin:          ctx.Stdin,
-			Stdout:         ctx.Stdout,
-			Stderr:         ctx.Stderr,
-			AuthorizedKeys: authKeys,
-			UpdateBehavior: &params.UpdateBehavior{
-				config.EnableOSRefreshUpdate(),
-				config.EnableOSUpgrade(),
-			},
+	if c.Placement != nil {
```

Comment:
I am sure @katco is asking for [idiomatic error flow indentation](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow), @hoenirvili